adapters/Mintegral/Public/Headers/GADMAdapterMintegralExtras.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+
+@interface GADMAdapterMintegralExtras : NSObject <GADAdNetworkExtras>
+
+
+/// Use this to mute for video ads. Must be set on each ad request.
+@property (nonatomic, assign) BOOL playVideoMute;
```

Comment:
To confirm, is the publisher required to set this **every time** when making an ad request? What will happen when the publisher haven't done so?

Reply:
No, it can be set when the publisher wants to mute the video,default is video unmuted